Problems with witty.er docker

Hi

I'm trying to create the docker image and fails. I have tried to do it in differents way with the same result: 

1.- Directly downloading the image from the docker repository:

$ singularity build wittyer.img docker://docker-general.dockerhub.illumina.com/wittyer:v0.3.1
INFO:    Starting build...
FATAL:   While performing build: conveyor failed to get: pinging docker registry returned: Get https://docker-general.dockerhub.illumina.com/v2/: dial tcp: lookup docker-general.dockerhub.illumina.com on 10.1.12.3:53: no such host
$ 

2.-  Using the docker-build.sh script:

misanchez@computer:~/Documents/Software/witty.er$ ./docker-build.sh 
Sending build context to Docker daemon  36.69MB
Step 1/11 : FROM microsoft/dotnet:2.0-sdk as builder
 ---> 9e06837225a4
Step 2/11 : WORKDIR /src
 ---> Using cache
 ---> b7ec75ca4f45
Step 3/11 : COPY . /src
 ---> Using cache
 ---> 54d169c6605b
Step 4/11 : RUN dotnet publish -f netcoreapp2.0 -r linux-x64 -c Release -o /output       && chmod +x /output/Wittyer
 ---> Running in 52a69375fc84
Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restoring packages for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj...
  Restoring packages for /src/Ilmn.Das.App.Wittyer/Ilmn.Das.App.Wittyer.csproj...
  Restoring packages for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj...
  Restoring packages for /src/Wittyer/Wittyer.csproj...
  Installing System.IO.UnmanagedMemoryStream 4.3.0.
  Installing System.IO.MemoryMappedFiles 4.3.0.
  Installing JetBrains.Annotations 11.1.0.
  Installing SourceLink.Embed.AllSourceFiles 2.8.1.
  Installing Newtonsoft.Json 12.0.1.
  Installing System.Collections.Immutable 1.5.0.
  Installing System.Composition.AttributedModel 1.1.0.
  Installing System.ComponentModel.Primitives 4.1.0.
  Installing System.Collections.Specialized 4.0.1.
  Installing System.ComponentModel 4.0.1.
  Installing System.ComponentModel.TypeConverter 4.1.0.
  Installing System.Diagnostics.TextWriterTraceListener 4.0.0.
  Installing System.ComponentModel.EventBasedAsync 4.0.11.
  Installing Microsoft.DotNet.PlatformAbstractions 1.0.3.
  Installing xunit.abstractions 2.0.2.
  Installing Microsoft.TestPlatform.ObjectModel 15.8.0.
  Installing Microsoft.Extensions.DependencyModel 1.0.3.
  Installing xunit.extensibility.core 2.4.0.
  Installing xunit.extensibility.execution 2.4.0.
  Installing System.Diagnostics.TraceSource 4.3.0.
  Installing xunit.runner.visualstudio 2.4.0.
  Installing Microsoft.TestPlatform.TestHost 15.8.0.
  Installing Microsoft.CodeCoverage 15.8.0.
  Installing Moq 4.9.0.
  Installing xunit.core 2.4.0.
  Installing xunit 2.4.0.
  Installing Microsoft.NET.Test.Sdk 15.8.0.
  Installing System.Linq.Queryable 4.3.0.
  Installing xunit.analyzers 0.10.0.
  Installing Castle.Core 4.3.1.
  Installing xunit.assert 2.4.0.
  Installing System.Xml.XPath.XDocument 4.0.1.
  Installing System.Diagnostics.StackTrace 4.0.1.
  Installing Microsoft.NETCore.DotNetHost 1.0.1.
  Installing Microsoft.CodeAnalysis.Common 1.3.0.
  Installing System.Security.Principal.Windows 4.0.0.
  Installing System.Security.Claims 4.0.1.
  Installing runtime.native.System.Net.Security 4.0.1.
  Installing Microsoft.NETCore.Jit 1.0.2.
  Installing Microsoft.NETCore.Windows.ApiSets 1.0.1.
  Installing Microsoft.NETCore.DotNetHostResolver 1.0.1.
  Installing dotnet-xunit 2.4.0-beta.1.build3958.
  Installing Microsoft.CodeAnalysis.CSharp 1.3.0.
  Installing Microsoft.CodeAnalysis.VisualBasic 1.3.0.
  Installing System.Net.NameResolution 4.0.0.
  Installing Microsoft.NETCore.App 1.0.0.
  Installing System.IO.MemoryMappedFiles 4.0.0.
  Installing System.Net.Security 4.0.0.
  Installing System.IO.UnmanagedMemoryStream 4.0.1.
  Installing Microsoft.NETCore.Runtime.CoreCLR 1.0.2.
  Installing System.Reflection.DispatchProxy 4.0.1.
  Installing System.Numerics.Vectors 4.1.1.
  Installing System.Net.WebHeaderCollection 4.0.1.
  Installing Libuv 1.9.0.
  Installing System.ComponentModel.Annotations 4.1.0.
  Installing Microsoft.VisualBasic 10.0.1.
  Installing System.Net.Requests 4.0.11.
  Installing System.Threading.Tasks.Parallel 4.0.1.
  Installing Microsoft.NETCore.DotNetHostPolicy 1.0.1.
  Installing System.IO.FileSystem.Watcher 4.0.0.
  Restore completed in 3.74 sec for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetAppHost 2.0.0.
  Installing runtime.any.System.Text.Encoding.Extensions 4.3.0.
  Installing runtime.any.System.Collections 4.3.0.
  Installing runtime.any.System.Diagnostics.Tracing 4.3.0.
  Installing runtime.unix.System.Private.Uri 4.3.0.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver 2.0.0.
  Installing runtime.any.System.Reflection.Primitives 4.3.0.
  Installing runtime.unix.System.Diagnostics.Debug 4.3.0.
  Installing runtime.linux-x64.Microsoft.NETCore.App 2.0.0.
  Installing runtime.any.System.Text.Encoding 4.3.0.
  Installing runtime.any.System.Reflection 4.3.0.
  Installing runtime.any.System.Globalization 4.3.0.
  Installing System.Private.Uri 4.3.0.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy 2.0.0.
  Installing runtime.unix.System.IO.FileSystem 4.3.0.
  Installing runtime.any.System.Runtime 4.3.0.
  Installing runtime.unix.System.Runtime.Extensions 4.3.0.
  Installing runtime.any.System.Runtime.InteropServices 4.3.0.
  Installing runtime.any.System.Resources.ResourceManager 4.3.0.
  Installing runtime.any.System.IO 4.3.0.
  Installing runtime.any.System.Threading.Tasks 4.3.0.
  Installing runtime.any.System.Runtime.Handles 4.3.0.
  Installing runtime.unix.System.Net.Primitives 4.3.0.
  Installing runtime.any.System.Threading.Timer 4.3.0.
  Installing runtime.unix.System.Console 4.3.0.
  Installing runtime.any.System.Diagnostics.Tools 4.3.0.
  Installing runtime.unix.System.Net.Sockets 4.3.0.
  Installing runtime.unix.Microsoft.Win32.Primitives 4.3.0.
  Installing runtime.any.System.Globalization.Calendars 4.3.0.
  Installing runtime.any.System.Reflection.Extensions 4.3.0.
/src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj : error NU1101: Unable to find package Ilmn.Das.Std.XunitUtils. No packages exist with this id in source(s): nuget.org [/src/Ilmn.Das.App.Wittyer.sln]
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer/obj/Ilmn.Das.App.Wittyer.csproj.nuget.g.props.
  Generating MSBuild file /src/Wittyer/obj/Wittyer.csproj.nuget.g.props.
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer/obj/Ilmn.Das.App.Wittyer.csproj.nuget.g.targets.
  Generating MSBuild file /src/Wittyer/obj/Wittyer.csproj.nuget.g.targets.
  Restore completed in 4.58 sec for /src/Ilmn.Das.App.Wittyer/Ilmn.Das.App.Wittyer.csproj.
  Restore completed in 4.58 sec for /src/Wittyer/Wittyer.csproj.
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer.Test/obj/Ilmn.Das.App.Wittyer.Test.csproj.nuget.g.props.
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer.Test/obj/Ilmn.Das.App.Wittyer.Test.csproj.nuget.g.targets.
  Restore failed in 4.64 sec for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj.
The command '/bin/sh -c dotnet publish -f netcoreapp2.0 -r linux-x64 -c Release -o /output       && chmod +x /output/Wittyer' returned a non-zero code: 1
misanchez@bigsuport:~/Documents/Software/witty.er$

3.- And the same problem if I try to build the container from the Dockerfile:

misanchez@bigsuport:~/Documents/Software/witty.er$ docker build --tag witty.er:1.0 .
Sending build context to Docker daemon  36.69MB
Step 1/10 : FROM microsoft/dotnet:2.0-sdk as builder
 ---> 9e06837225a4
Step 2/10 : WORKDIR /src
 ---> Using cache
 ---> b7ec75ca4f45
Step 3/10 : COPY . /src
 ---> Using cache
 ---> 54d169c6605b
Step 4/10 : RUN dotnet publish -f netcoreapp2.0 -r linux-x64 -c Release -o /output       && chmod +x /output/Wittyer
 ---> Running in f47fc4a63c2c
Microsoft (R) Build Engine version 15.7.179.6572 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restoring packages for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj...
  Restoring packages for /src/Ilmn.Das.App.Wittyer/Ilmn.Das.App.Wittyer.csproj...
  Restoring packages for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj...
  Restoring packages for /src/Wittyer/Wittyer.csproj...
  Installing System.IO.UnmanagedMemoryStream 4.3.0.
  Installing System.IO.MemoryMappedFiles 4.3.0.
  Installing Newtonsoft.Json 12.0.1.
  Installing System.Collections.Immutable 1.5.0.
  Installing System.Composition.AttributedModel 1.1.0.
  Installing JetBrains.Annotations 11.1.0.
  Installing SourceLink.Embed.AllSourceFiles 2.8.1.
  Installing System.Xml.XPath.XDocument 4.0.1.
  Installing System.Diagnostics.StackTrace 4.0.1.
  Installing Microsoft.NETCore.DotNetHost 1.0.1.
  Installing Microsoft.CodeAnalysis.Common 1.3.0.
  Installing System.Security.Principal.Windows 4.0.0.
  Installing System.Security.Claims 4.0.1.
  Installing runtime.native.System.Net.Security 4.0.1.
  Installing Microsoft.NETCore.Windows.ApiSets 1.0.1.
  Installing Microsoft.NETCore.Jit 1.0.2.
  Installing Microsoft.NETCore.DotNetHostResolver 1.0.1.
  Installing dotnet-xunit 2.4.0-beta.1.build3958.
  Installing runtime.any.System.Collections 4.3.0.
  Installing Microsoft.CodeAnalysis.VisualBasic 1.3.0.
  Installing runtime.any.System.Text.Encoding.Extensions 4.3.0.
  Installing Microsoft.CodeAnalysis.CSharp 1.3.0.
  Installing runtime.any.System.Diagnostics.Tracing 4.3.0.
  Installing Microsoft.NETCore.App 1.0.0.
  Installing System.Net.NameResolution 4.0.0.
  Installing runtime.unix.System.Private.Uri 4.3.0.
  Installing System.Net.Security 4.0.0.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetAppHost 2.0.0.
  Installing Microsoft.NETCore.Runtime.CoreCLR 1.0.2.
  Installing runtime.any.System.Reflection.Primitives 4.3.0.
  Installing Microsoft.NETCore.DotNetHostPolicy 1.0.1.
  Installing runtime.any.System.Text.Encoding 4.3.0.
  Installing Libuv 1.9.0.
  Installing System.IO.UnmanagedMemoryStream 4.0.1.
  Installing runtime.unix.System.Diagnostics.Debug 4.3.0.
  Installing runtime.any.System.Reflection 4.3.0.
  Installing runtime.any.System.Globalization 4.3.0.
  Installing System.IO.MemoryMappedFiles 4.0.0.
  Installing System.Threading.Tasks.Parallel 4.0.1.
  Installing System.Private.Uri 4.3.0.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostResolver 2.0.0.
  Installing runtime.any.System.IO 4.3.0.
  Installing runtime.any.System.Runtime.InteropServices 4.3.0.
  Installing runtime.unix.System.IO.FileSystem 4.3.0.
  Installing System.Numerics.Vectors 4.1.1.
  Installing runtime.any.System.Threading.Tasks 4.3.0.
  Installing System.ComponentModel.Annotations 4.1.0.
  Installing runtime.linux-x64.Microsoft.NETCore.App 2.0.0.
  Installing System.Net.Requests 4.0.11.
  Installing System.ComponentModel 4.0.1.
  Installing System.Net.WebHeaderCollection 4.0.1.
  Installing System.IO.FileSystem.Watcher 4.0.0.
  Installing System.Reflection.DispatchProxy 4.0.1.
  Installing runtime.linux-x64.Microsoft.NETCore.DotNetHostPolicy 2.0.0.
  Installing Microsoft.VisualBasic 10.0.1.
  Installing runtime.unix.System.Runtime.Extensions 4.3.0.
  Installing runtime.any.System.Runtime.Handles 4.3.0.
  Installing runtime.any.System.Runtime 4.3.0.
  Installing runtime.any.System.Resources.ResourceManager 4.3.0.
  Restore completed in 3.69 sec for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj.
  Generating MSBuild file /src/Wittyer/obj/Wittyer.csproj.nuget.g.props.
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer/obj/Ilmn.Das.App.Wittyer.csproj.nuget.g.props.
  Generating MSBuild file /src/Wittyer/obj/Wittyer.csproj.nuget.g.targets.
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer/obj/Ilmn.Das.App.Wittyer.csproj.nuget.g.targets.
  Restore completed in 4.1 sec for /src/Wittyer/Wittyer.csproj.
  Restore completed in 4.1 sec for /src/Ilmn.Das.App.Wittyer/Ilmn.Das.App.Wittyer.csproj.
  Installing System.Collections.Specialized 4.0.1.
  Installing System.ComponentModel.Primitives 4.1.0.
  Installing System.ComponentModel.TypeConverter 4.1.0.
  Installing System.ComponentModel.EventBasedAsync 4.0.11.
  Installing System.Diagnostics.TextWriterTraceListener 4.0.0.
  Installing Microsoft.DotNet.PlatformAbstractions 1.0.3.
  Installing xunit.abstractions 2.0.2.
  Installing Microsoft.TestPlatform.ObjectModel 15.8.0.
  Installing Microsoft.Extensions.DependencyModel 1.0.3.
  Installing xunit.extensibility.core 2.4.0.
  Installing xunit.extensibility.execution 2.4.0.
  Installing System.Diagnostics.TraceSource 4.3.0.
  Installing Microsoft.TestPlatform.TestHost 15.8.0.
  Installing Microsoft.CodeCoverage 15.8.0.
  Installing xunit.core 2.4.0.
  Installing xunit.analyzers 0.10.0.
  Installing xunit.runner.visualstudio 2.4.0.
  Installing Moq 4.9.0.
  Installing xunit 2.4.0.
  Installing xunit.assert 2.4.0.
  Installing Microsoft.NET.Test.Sdk 15.8.0.
  Installing System.Linq.Queryable 4.3.0.
  Installing Castle.Core 4.3.1.
  Installing runtime.unix.System.Net.Sockets 4.3.0.
  Installing runtime.unix.System.Console 4.3.0.
  Installing runtime.unix.System.Net.Primitives 4.3.0.
  Installing runtime.any.System.Threading.Timer 4.3.0.
  Installing runtime.any.System.Diagnostics.Tools 4.3.0.
  Installing runtime.any.System.Globalization.Calendars 4.3.0.
  Installing runtime.unix.Microsoft.Win32.Primitives 4.3.0.
  Installing runtime.any.System.Reflection.Extensions 4.3.0.
/src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj : error NU1101: Unable to find package Ilmn.Das.Std.XunitUtils. No packages exist with this id in source(s): nuget.org [/src/Ilmn.Das.App.Wittyer.sln]
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer.Test/obj/Ilmn.Das.App.Wittyer.Test.csproj.nuget.g.props.
  Generating MSBuild file /src/Ilmn.Das.App.Wittyer.Test/obj/Ilmn.Das.App.Wittyer.Test.csproj.nuget.g.targets.
  Restore failed in 5.38 sec for /src/Ilmn.Das.App.Wittyer.Test/Ilmn.Das.App.Wittyer.Test.csproj.
The command '/bin/sh -c dotnet publish -f netcoreapp2.0 -r linux-x64 -c Release -o /output       && chmod +x /output/Wittyer' returned a non-zero code: 1
misanchez@bigsuport:~/Documents/Software/witty.er$

Can you help me, please?

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,8 @@
 FROM microsoft/dotnet:2.0-sdk as builder
 WORKDIR /src
 COPY . /src
-RUN dotnet publish -f netcoreapp2.0 -r linux-x64 -c Release -o /output \
+RUN cd Wittyer \
+      && dotnet publish -f netcoreapp2.0 -r linux-x64 -c Release -o /output \
       && chmod +x /output/Wittyer
       
 FROM microsoft/dotnet:2.0.9-runtime

